WORKSPACE

```diff
@@ -14,7 +14,7 @@ envoy_dependencies()
 
 load("//bazel:repositories_extra.bzl", "envoy_dependencies_extra")
 
-envoy_dependencies_extra()
+envoy_dependencies_extra(ignore_root_user_error=True)
```

Comment:
Why do we need this?  Consider leaving a comment about it.

Reply:
Good point. This is needed because the updated version of `rules_python` in Bazel returns an error if it is run as the root user. Since we run this in a Docker container, we are running as root making this a problem. Here's the relevant info: https://github.com/bazelbuild/rules_python/tree/565a531d167a6f066ff555d8831a30a38dacb74c/tests/integration/ignore_root_user_error